tldw_Server_API/app/api/v1/endpoints/admin.py

```
@@ -2224,138 +2111,40 @@ async def delete_user_override(user_id: int, permission_id: int, db=Depends(get_
-                ORDER BY p.name
-                """,
-                (role_id,),
-            )
-            rows2 = await cur2.fetchall()
-            names = [str(x[0]) for x in rows2]
-
-        tool_prefix = 'tools.execute:'
-        tool_permissions = [n for n in names if n.startswith(tool_prefix)]
-        permissions = [n for n in names if not n.startswith(tool_prefix)]
-        all_permissions = sorted(set(tool_permissions) | set(permissions))
+        repo = AuthnzRbacRepo()
```

Ruff (0.14.6)

Undefined name `AuthnzRbacRepo`

(F821)